### ISSUE TITLE
initYTPlayer called before setSrc when using video.js dynamically

### ISSUE THREAD

**Issue by grzegorzjudas** [eXon/videojs-youtube#321]
Hi @eXon, great job with the plugin,

however, I'm using video.js with angular and a lot of server-backed data and the only way for me to init the player is dynamically (by calling *videojs(element, options, readyFunction*).

That being said, the video did not load correctly - plugin successfully loaded youtube iframe API script, but then, it went straight to initYTPlayer function and tried to register the player with youtube video ID which didn't exist yet. That id, as well as the whole source url parsing, is called in constructor, which, for some reason, is called afterwards. Easily trackable by setting up breakpoints in both *initYTPlayer* and *constructor*.

Here's an example:
```javascript
videojs('#player', {
    controls: true,
    sources: [ { src: 'someYoutubeUrl', type: 'video/youtube' } ],
    techOrder: [ 'youtube', 'html5' ]
});
```

Or maybe I'm doing something wrong, but I didn't find any example in the repo that uses this method of inititalizing video.js.

**Comment by eXon** [comment]
Their is a known bug about it I need to fix. A fix will be released soon.

**Comment by eXon** [comment]
Should be fixed in 2.0.1
